code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -196,7 +196,8 @@ update_label("John Doe", "Clowny")
 	icon_state = "centcom"
 	registered_name = "Emergency Response Team Commander"
 	assignment = "Emergency Response Team Commander"
-	access = list(access_sec_doors, access_engine, access_medical)
+	New()

```

Comment:
Full typepaths please. also ```access = get_all_accesses()``` wasn't quite what you were asked to do.
Miauw meant make an ```/obj/item/weapon/card/id/ert``` which has ```access = get_all_accesses()``` on it (or some other form of all access) and just have ```/obj/item/weapon/card/id/ert/Security```, ```/obj/item/weapon/card/id/ert/Engineer``` etc.